### DISCUSSION TITLE
Is there any way to get the hostname request inside the function trigger after adding data in the auth.users table?

### DISCUSSION THREAD

**Discussion by thisisbintang** [supabase/supabase#12360] [Questions]
I have tried to get the request headers information inside the trigger function after adding data in the public schema table, but in the auth schema it is missing. Is there any way to get it?

**Comment by GaryAustin1** [comment]
I don’t believe so as it is PostgREST API for database operations that sets it up.  Gotrue does not set that info anywhere in the database.
